.evergreen/run-tests.sh

```diff
@@ -21,6 +22,12 @@ export PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig"
 # override where we look for libmongoc
 export LD_LIBRARY_PATH="${INSTALL_DIR}/lib"
 export DYLD_LIBRARY_PATH="${INSTALL_DIR}/lib"
+export DEVELOPER_DIR=/Applications/Xcode10.1.app
```

Comment:
how are we testing swift 5 if we are always using xcode 10.1?

Reply:
This is a little misleading I now see. I'm pointing at a `DEVELOPER_DIR` for xcode 10.1, so that we have the right OSX SDK in place (not swift), but we're actually using the Swift 5 toolchain downloaded by swiftenv.